.github/workflows/test.yml

```diff
@@ -17,6 +17,7 @@ jobs:
     strategy:
       matrix:
         node-version:
+          - 10.18.0
           - 10
```

Comment:
do we need both? Maybe testing with the lowest version we officially support will suffice?

Reply:
I could be convinced either way. It's possible for a change to be introduced in a later version that could cause a problem, but probably a bug in nice if it's breaking.

Reply:
I'd just add `10.18.0` and trust the node folks that they don't break anything in maintenance version. It's very unlikely. It will speed up our tests, as GitHub Actions limit the amount of parallel builds

Reply:
i pulled `10` out of the list

Reply:
I still see it?

https://github.com/semantic-release/semantic-release/blob/engine-strict/.github/workflows/test.yml#L21

Reply:
somehow managed to not push the change 🤦🏼 should be there now